prime-router/src/main/kotlin/azure/CheckFunction.kt

```diff
@@ -17,8 +17,6 @@ import gov.cdc.prime.router.azure.db.enums.SettingType
 import gov.cdc.prime.router.common.BaseEngine
 import gov.cdc.prime.router.common.JacksonMapperUtilities
 import gov.cdc.prime.router.credentials.RestCredential
-import gov.cdc.prime.router.tokens.AuthenticatedClaims
```

Comment:
Let's unstage the Kotlin changes from this changeset since they're not directly related to the link stuff :)

Reply:
I wouldn't have been able to do the commit without the ktlinting that someone forgot to do.